### ISSUE TITLE
Zentyal is alive or not?

### ISSUE THREAD

**Issue by Neustradamus** [zentyal/zentyal#2085]
Zentyal is alive or not?

You can see PRs here:
- https://github.com/zentyal/zentyal/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc

You can see Issues here:
- https://github.com/zentyal/zentyal/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

**Comment by Zentyal-Admin** [comment]
Very much alive. Currently scaling to be US-centric. Much more to come.

**Comment by Facsimiler** [comment]
Hi @Zentyal-Admin!

Are you taking over supporting Zentyal? What is the plan? What is the status? This is more than curiosity: Zentyal is very important to me, and if it's no longer a viable product, then I need to switch to something that is. So anything you can provide in the way of an update would be very much appreciated.

If you need assistance maintaining Zentyal, then I'm sure that would be useful to add, too. (I would be prepared to do some bug-fixing.)

Thanks!

**Comment by bitjbullock** [comment]
Bump. I've been talking to clients about Zentyal as a possible replacement for Windows Servers. If its not actively being supported then I need to re-adjust my recommendations

**Comment by gallogabor** [comment]
Nothing happens around Zentyal for months now. Many bug reports made, multiple pull requests appeared, but no one takes care about those.

It's like Zentyal's development suddenly stopped...

I saw the comment above that the project is alive, but that comment created on January 13, 2022, and the date is September 19, 2022 now, and nearly nothing happened...

I'm worried about Zentyal now.

**Comment by peptoniet** [comment]
@Zentyal-Admin commented on January they were scaling to be US-centric...  I guess it is a long journey to the US...  Is it very much to ask for some information about current status? not to mention any commitment...
It is sad.  A nice project...
If you (the developers) could be more transparent, the users could make an informed decision, weather to fork, or move to other projects.
Please, speak up, with complete and true information.  Don't be silent, or give vague answers.  We beg you.  We trusted you for a long time.

**Comment by progamer562** [comment]
im with you guys this project is very dead and hard to move away

**Comment by peptoniet** [comment]
And now we have a very serious problem with Windows 11 22H2, not being able to join domain...  Take a look at pending bugs.

**Comment by progamer562** [comment]
i know i saw

**Comment by peptoniet** [comment]
It is sad, the worst part of it is the silence...  This is not the spirit of open source.
They do work... on hidden repositories.
Well, good luck with your projects.  I hope you will achieve success.

**Comment by brunolorente** [comment]
Hi everyone,

Zentyal is still alive, and we are working on the fix for several bugs, which will be released this month and the next. In parallel, we have started to work on the new **Zentyal 8.0 (based on Ubuntu 22 with all that involves)** that will be released the next May.

We are closing this.

**Comment by Zentyal-Admin** [comment]
@peptoniet so sorry to see you go. While @brunolorente and the team have been working hard on the new release and bug fixes in Spain, I have been working with the marketing team in New York to get ready for 2023. We are about to introduce our new partner program which enables us to pay for referrals but we needed to get nice and stable again after repositioning the company for larger scale. We have big plans but it has taken longer than we had hoped because our team is small but we are growing. Sorry again for silence here. I had not been monitoring these threads but we will do better with our attention to our customer base. Best of luck with everything for you.

**Comment by progamer562** [comment]
will i hope the release supports docker thats a must for me

**Comment by Neustradamus** [comment]
If Zentyal is not dead, it is possible to merge PRs?

**Comment by gallogabor** [comment]
I think the closing of this ticket was a mistake. The question asked over a year ago now more relevant that ever.

A project with last commit made 8 months ago is minimum stale, if not dead...
The communication from the dev team also stale, if not dead...
There are active serious problems without any fix, or at least a promise to fix...
There are no any updates on roadmap, progress with the above mentioned changes, etc.

I loved and used Zentyal from version 4, but I think I cannot build any production system on this project/software at least in the near future.
I wish this project soon became a phoenix and rises from its ashes!

**Comment by progamer562** [comment]
DOCKER DOCKER

**Comment by wirdo89** [comment]
> I think the closing of this ticket was a mistake. The question asked over a year ago now more relevant that ever.
> 
> A project with last commit made 8 months ago is minimum stale, if not dead... The communication from the dev team also stale, if not dead... There are active serious problems without any fix, or at least a promise to fix... There are no any updates on roadmap, progress with the above mentioned changes, etc.
> 
> I loved and used Zentyal from version 4, but I think I cannot build any production system on this project/software at least in the near future. I wish this project soon became a phoenix and rises from its ashes!

I'm on the same boat. I want to deploy a new server for one of our customers with Zentyal but it very much seem dead to me.
They haven't posted anything on their website since 2021.

**Comment by peptoniet** [comment]
> Hi everyone,
> 
> Zentyal is still alive, and we are working on the fix for several bugs, which will be released this month and the next. In parallel, we have started to work on the new **Zentyal 8.0 (based on Ubuntu 22 with all that involves)** that will be released the next May.
> 
> We are closing this.

Hi,

Just to restore some faith, if possible.  Where are these fixed bugs, supposedly released in a month time starting October 13, 2022...?  No updates have been made to the repository...  for 8 months.
Why don't you just tell the truth?
You are under no obligation to post anything to a public repo.  Just don't say you will.  It hurts the open source community.
Was this an open source project from the start, or just a misuse of the open source world...?  Care to explain why your users are contributing to hidden repos, but not the public ones...?
How about the pull requests...?

Your actions, not your words, dictate the project is dead.

Thank you very much.

**Comment by dzidek23** [comment]
All that activity on a closed issue is fascinating.
I, just like many others, am waiting for news, updates or even a smallest change that would suggest Zentyal isn't dead. 

I'm far from making the kind of comments @peptoniet has made. I was well aware that Zentyal is a project supported by a small team, one that is up against Microsoft hegemony, a project which could stall and die. 

Current functionality and the fact that one can install Zentyal 7 on Ubuntu 20.04 allowed me to make an informed choice to make Zentyal a domain controller for a small intranet. It serves well for DHCP, file shares, AD domain, Radius auth for wireless and VPN. I'm far from being worried about the 2025 when Ubuntu 20.04 will hit EOL or Jun 2024 when Windows 21H2 stops updating.

I will (can) patiently wait for May and the mentioned Zentyal 8.

However, looking at this repo's activity, I'm not surprised why people are anxious. Pull requests created in August 2021, with some security updates, surely should have been reviewed and implemented ages ago, to benefit all users and the project alike.
It would at least show that you are still there. Updated your website if that's the only thing you can do to show you are alive. 

Lastly, I'm here to thank you @Zentyal-Admin @brunolorente for your project and all efforts you make to maintain it. Zentyal is easy to install and manage, pleasing in its design and although it could do with a few improvements, it beats any other distro offering Windows Server functionality.

Keep up the good work, be a tad more interactive and we are waiting for Zentyal 8.
Cheerio!

**Comment by peptoniet** [comment]
I feel I have to answer to some of these affirmations, as I have been mentioned on the post (no hard feelings from me).

Zentyal is a project supported by a small team, because they wanted to be this way.  How? Not contributing to the general repository but, allegedly, to private ones.  Allegedly, Zentyal is (was) being developed on private repositories and then published to the official ones, when the small group of developers wanted.  This behaviour only contributes to keep the team small.  There is nothing humble or positive in being a small team.  They are a small team because they did not open their project to the community.  Publishing source code is not Open Source project...

5 contributions in private repositories Mar 2
71 contributions in private repositories Feb 1 – Feb 28
19 contributions in private repositories Jan 5 – Jan 30
4 contributions in private repositories Dec 2 – Dec 19
65 contributions in private repositories Nov 2 – Nov 29
95 contributions in private repositories Oct 1 – Oct 29
61 contributions in private repositories Sep 4 – Sep 29 

Why, then, Zentyal has not been forked? There are two reasons why a project does not fork: Main branch is good enough, active and open to the community. Second, project is too flawed to benefit from branching.  Make your choice.

"it beats any other distro offering Windows Server functionality".  Literally, any other distro with similar functionality is way better supported, with more regular updates and more functionality than Zentyal.  I am really curious how Zentyal "beats" ALL other distros...

Certanly, it does not beat them in support... If you restore from a Zentyal backup, be careful.  If you upgrade Zentyal, be careful.  I advice anyone to take a tour around Zentyal forum...  Lots of opening eyes questions there.

At the very time of writing: Univention updated 10 hours ago...  Nethserver updated yesterday... just a sample.

Good day.

**Comment by dzidek23** [comment]
Ditto on some things, but..

Just been playing with UCS and I see that although it looks much more, how to say this, polished it is far from ideal. Has tones of options, but it talks to Univention server bit too much for my liking. Can't get Radius to install either...

Let's leave it there, we shall see what is going to happen with Zentyal in few months.

Cheerio!


8 Mar 2023 07:58:57 peptoniet ***@***.***>:

**Comment by Rubarius** [comment]
What analogues are there besides Univention and Netserver? I didn't like them very much. I need a new version of samba and AD.

**Comment by Rubarius** [comment]
May 16, already the middle of the month...

**Comment by dzidek23** [comment]
@peptoniet not wanting to say you were right, but here we are. Mid-may and no sign of an update or even communication. The only thing I noticed that has changed is some 'customer testimonials / use cases' on the front page.

I will treat this as a dead project and plan for migration, saying this I'm far from panic mode. Ubuntu 20.04 will be supported until 2025, Windows 10 22H2 joined the domain without any problems. Long time to migrate, maybe even to something cloud based.

It is a real shame it ends here, Zentyal is a good product and did have the potential to pull people out of the hands of Microsoft.

**Comment by peptoniet** [comment]
Well.. to be fair...  they are, allegedly, working on it...  (I want to believe so...) because if you look at the developers' activity, well... they are contributing to private repositories...  back to my idea of what is NOT open source.  Also, some pull requests have been merged, but only the ones created by the _small team_ of developers... the ones proposed by the community are still sleeping a dream within a dream...  </ironic mode off>
Univention is a powerful beast... I live happy now.

_April, come she will_
_When streams are ripe and swelled with rain_
_May, she will stay_
_Resting in my arms again._

_Simon & Garfunkel._

**Comment by Pharao01** [comment]
Still hoping that Zentyal 8 will be officially announced soon!
Keep going!

**Comment by mhbeyle** [comment]
A project of this type that has not been updated for two years or more is a dead project.

So far we have been patching bugs and security updates, but it is time for change.

Is there any news about the creation of some kind of fork by someone or do we have to move to other solutions like nethserver or UCS?

**Comment by dzidek23** [comment]
@mhbeyle I think this thread is a good example of the project status. 
We are planning migrating off Zentyal in the very near future.

**Comment by Rubarius** [comment]
Looks like this is the end. Migrating to Unvention.

**Comment by mhbeyle** [comment]
Is there an easy way to migrate existing installations to UCS or is it better to start from scratch?
There are quite complex installations to move around here.

@peptoniet I think you did the migration... Did you have any problems? Did you get help from a guide?

**Comment by peptoniet** [comment]
Strange... out of curiosity, I tested "samba-tool dbcheck --cross-ncs" on some of my Univention...  they show no problem...  Why was Zentyal showing problems on ALL of my domains...?

**Comment by trvaa** [comment]
The may explain
https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

**Comment by trvaa** [comment]
> Zentyal is alive or not?
> 
> You can see PRs here:
> 
> * https://github.com/zentyal/zentyal/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc
> 
> You can see Issues here:
> 
> * https://github.com/zentyal/zentyal/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

This may explain

https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp

**Comment by Zentyal-Admin** [comment]
Hi everyone. This thread was not being closely monitored, but I would like to assure you all that Zentyal is very much alive, supported by a small development team in Spain but is now US owned and headquartered in New York. The release of 8.0 was planned for 2023 midyear but was delayed due to installer issues, and a new installer had to be created, pushing the 8.0 release back to December 2023. The new installer is almost ready, along with many bug fixes. The development team leader stated that monthly releases are planned from December onward. We will be monitoring this feed more closely as we ramp up our efforts to deliver a stable product that we can all be proud of. Zentyal has been a well-supported product, and although the team is small, we plan to increase its footprint globally while building on a solid foundation.

**Comment by peptoniet** [comment]
In june, may is december.  Good start, folks.  U.S.A. never dissappoints.  Have fun. ))))))

**Comment by mhbeyle** [comment]
> As usual, I am ashamed to be spanish...

Don't go too far now. It's the same everywhere you go.

**Comment by arusconi** [comment]
Hi ! Sorry, did you notice the publication date of the news? : 
06/02/2019 ...

El 9/6/23 a las 11:24, trvaa escribió:
> The may explain
> https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp 
> <https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp>

Best regards.

Andrés

**Comment by peptoniet** [comment]
> Hi ! Sorry, did you notice the publication date of the news? : 06/02/2019 ... El 9/6/23 a las 11:24, trvaa escribió:
> The may explain https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp <https://www-heraldo-es.translate.goog/noticias/aragon/zaragoza/2019/02/06/audiencia-zaragoza-condena-dos-ejecutivos-empresa-zentyal-administracion-desleal-1291185-2261126.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp>
> Best regards. Andrés

Yeah... It makes me even more sad...  to have overlooked such news

**Comment by rubenszolt** [comment]
> In addition, The 7.1 release (updating the installer and several bug fixes) is being finalized for distribution by the end of June 2023.

Any update related to 7.1 release?

**Comment by peptoniet** [comment]
> > In addition, The 7.1 release (updating the installer and several bug fixes) is being finalized for distribution by the end of June 2023.
> 
> Any update related to 7.1 release?

I think you are being very picky... In my time zone, there are still 4 hour and 50 minutes of June...  Just be patient... ))))

**Comment by rubenszolt** [comment]
> 

I've been very patient for a long time, but I really want something to happen here. And I think it's July in all time zones now :-))))

**Comment by peptoniet** [comment]
> > 
> 
> I've been very patient for a long time, but I really want something to happen here. And I think it's July in all time zones now :-))))

You are, indeed, right.  Definetly, this is a scam.  Or at least, vaporware.

**Comment by rubenszolt** [comment]
> 

I would have been surprised if it was really released at the end of June, I think everything here is dead, they just don't know about it yet. I don't think there will be an 8.0 release, just dreams.

**Comment by Pharao01** [comment]
I'm still convinced that they're working to the goal of new releases. It is a small team with small resources and lack of resources often means delays and unpredictable hurdles.
And think about this: what benefit would they have in telling us that there will be a new release when there isn't a spark of a chance for it.
If Zentyal would be dead already why not just telling so and bothering to answer posts and unpleasant questions in this thread.

**Comment by peptoniet** [comment]
> I'm still convinced that they're working to the goal of new releases. It is a small team with small resources and lack of resources often means delays and unpredictable hurdles. And think about this: what benefit would they have in telling us that there will be a new release when there isn't a spark of a chance for it. If Zentyal would be dead already why not just telling so and bothering to answer posts and unpleasant questions in this thread.

Investors...? In U. S. A.?

**Comment by rubenszolt** [comment]
I think the biggest problem here is not the delay, but the lack of communication. It can be developed with a small team, many people do it, but at least in a community. The problem is that by the time something happens here, it might already be too late. The solution would be to open up the project to the community, more people could help. 
They can't develop without money either, and I think that's the situation now.

**Comment by Zentyal-Admin** [comment]
I PROMISE YOU ALL IT IS ALMOST READY TO DEPLOY. Yes, sorry for the delay. We had to entirely build a new installer which delayed our V8.0 release which is now at the end of 2023. I won't commit to a month because it has been a little unpredictable. The developers tell us June and here it is July but they are doing a great job and it is coming this week. Our lead developer said "This week we will be ready to release the new installer based on the last ubuntu 20 and bug fixes in the 7.1 release." Then once we know we are stable and everyone is happy we are working diligently toward 8.0. We have been expanding to fund the new development and we are on track.

**Comment by rubenszolt** [comment]
Thank you for the feedback, it is important for us who love Zentyal and have been waiting for any feedback for a long time. I hope that the 7.1 release will arrive this week, with a lot of bug fixes. :-)
And guys, don't forget that communication helps a lot even in difficult situations.

**Comment by progamer562** [comment]
docker?

**Comment by fhatz** [comment]
I think the project is dead. Only the facebook-page has new updates, nothing to see on the official website. It seems somebody is trying to keep the project alive or to revitalize it. It is sad...I liked the project very much, for many years.
Planning moving to UCS....

**Comment by peptoniet** [comment]
I'm confused: last time I checked, a week had 7 days.

**Comment by progamer562** [comment]
> I think the project is dead. Only the facebook-page has new updates, nothing to see on the official website. It seems somebody is trying to keep the project alive or to revitalize it. It is sad...I liked the project very much, for many years. Planning moving to UCS....

whats ucs?

**Comment by peptoniet** [comment]
> > I think the project is dead. Only the facebook-page has new updates, nothing to see on the official website. It seems somebody is trying to keep the project alive or to revitalize it. It is sad...I liked the project very much, for many years. Planning moving to UCS....
> 
> whats ucs?

Univention corporate server.
www.univention.com
I have migrated 16 domains from Zentyal to Univention.

**Comment by dzidek23** [comment]
> I PROMISE YOU ALL IT IS ALMOST READY TO DEPLOY. Yes, sorry for the delay. We had to entirely build a new installer which delayed our V8.0 release which is now at the end of 2023. I won't commit to a month because it has been a little unpredictable. The developers tell us June and here it is July but they are doing a great job and it is coming this week. Our lead developer said "This week we will be ready to release the new installer based on the last ubuntu 20 and bug fixes in the 7.1 release." Then once we know we are stable and everyone is happy we are working diligently toward 8.0. We have been expanding to fund the new development and we are on track.

@Zentyal-Admin why wouldn't you share release candidate with us? As said before, testing would be much easier with the community involvement than without it.

**Comment by peptoniet** [comment]
> > I PROMISE YOU ALL IT IS ALMOST READY TO DEPLOY. Yes, sorry for the delay. We had to entirely build a new installer which delayed our V8.0 release which is now at the end of 2023. I won't commit to a month because it has been a little unpredictable. The developers tell us June and here it is July but they are doing a great job and it is coming this week. Our lead developer said "This week we will be ready to release the new installer based on the last ubuntu 20 and bug fixes in the 7.1 release." Then once we know we are stable and everyone is happy we are working diligently toward 8.0. We have been expanding to fund the new development and we are on track.
> 
> @Zentyal-Admin why wouldn't you share release candidate with us? As said before, testing would be much easier with the community involvement than without it.

I also want to know the answer to this question.  Zentyal tema arguments about how small the developers team is, but they would not publish the ongoing work, for others to review and colaborate.

**Comment by Rubarius** [comment]
"i promise you all it is almost ready to deploy", until December 2023, almost 5 months. Very strange.

**Comment by pitfiend** [comment]
> I PROMISE YOU ALL IT IS ALMOST READY TO DEPLOY. Yes, sorry for the delay. We had to entirely build a new installer which delayed our V8.0 release which is now at the end of 2023. I won't commit to a month because it has been a little unpredictable. The developers tell us June and here it is July but they are doing a great job and it is coming this week. Our lead developer said "This week we will be ready to release the new installer based on the last ubuntu 20 and bug fixes in the 7.1 release." Then once we know we are stable and everyone is happy we are working diligently toward 8.0. We have been expanding to fund the new development and we are on track.

blablabla... mid August and still nothing. The worst is to build some install script for a particular release of a particular distro. This is a recipe for disaster. Every bit here is perfectly available on the repos, the sole thing we will miss is a fancy web interface. You are selling smoke or flavored snake oil.

Your commitment with the Open Source Movement is dubious and sneaky.

**Comment by Pharao01** [comment]
To have an update here: I just upgraded to 7.1 which is available I think fo a bit over a month now 😊
So, looking frward to 8.0 🤞

**Comment by Neustradamus** [comment]
We are in December.

**Comment by progamer562** [comment]
die this project isnt working now

**Comment by lightsabata** [comment]
@Zentyal-Admin, any update or ETA about 8.0 ?

**Comment by FlexibleToast** [comment]
So... With this issue being closed, it's 2024 and no Zentyal 8, does that mean it is dead?

**Comment by peptoniet** [comment]
It's dead.  Nobody needs the death certificate from the admin or developers.  It is not breathing, thus is dead.
I strongly advise everybody to migrate, as soon as possible, to another solution.
This project is not even worth forking.  It is flawed all around.  I know what I am talking about.
As admin or developers have not acknowledged the existance of "hidden" repos, we must believe the last interation with the code was six months ago...  DEAD.

**Comment by lightsabata** [comment]
So sad… @peptoniet do you know some similar project thar your have tried and recommend ?

**Comment by peptoniet** [comment]
> So sad… @peptoniet do you know some similar project that your have tried and recommend ?

As for my clients (20 domains, some > 1.000.000 € / month invoicing enterprises) I have migrated to Univention (www.univention.com).  It has a tool (AD Takeover) to seamlessly migrate a domain from Windows, Zentyal, Samba, etc. to Univention.  Tried from Zentyal and from Windows.

Univention puts Zentyal to BIG shame in updates (every 1 or 2 weeks), API, tools, interface, plugins, automation...  with a longer lifetime and experience.  I had absolutely NO problems with Univention so far.

**Comment by Neustradamus** [comment]
Since several years, I have promoted Zentyal (formerly eBox), on my social networks - that I propose you to follow, links in my bio account - I have already promoted @NethServer and @Univention UCS.

**Comment by mhbeyle** [comment]
> Since several years, I have promoted Zentyal (formerly eBox), on my social networks - that I propose you to follow, links in my bio account - I have already promoted @NethServer and @univention UCS.

I think nethserver is currently the best alternative to zentyal if the goal is to use samba ad over an infrastructure of ms windows machines.

However, the installation (also in univention) has to be done cleanly. Forget about transferring information from zentyal to nethserver/univention. It is not worth it in working hours.

**Comment by peptoniet** [comment]
> > Since several years, I have promoted Zentyal (formerly eBox), on my social networks - that I propose you to follow, links in my bio account - I have already promoted @NethServer and @univention UCS.
> 
> I think nethserver is currently the best alternative to zentyal if the goal is to use samba ad over an infrastructure of ms windows machines.
> 
> However, the installation (also in univention) has to be done cleanly. Forget about transferring information from zentyal to nethserver/univention. It is not worth it in working hours.

I disagree.  If you change domain, you will have to manually include all your workstations in the new domain.  I have compilated a toolbox with Ansible and bash, which makes de transition seamless.  I cannot afford to migrate hundreds of machines to another domain.

What was taking so many hours in your experience...?

**Comment by mhbeyle** [comment]
> > > Since several years, I have promoted Zentyal (formerly eBox), on my social networks - that I propose you to follow, links in my bio account - I have already promoted @NethServer and @univention UCS.
> > 
> > 
> > I think nethserver is currently the best alternative to zentyal if the goal is to use samba ad over an infrastructure of ms windows machines.
> > However, the installation (also in univention) has to be done cleanly. Forget about transferring information from zentyal to nethserver/univention. It is not worth it in working hours.
> 
> I disagree. If you change domain, you will have to manually include all your workstations in the new domain. I have compilated a toolbox with Ansible and bash, which makes de transition seamless. I cannot afford to migrate hundreds of machines to another domain.
> 
> What was taking so many hours in your experience...?

Yes, that is correct, but in my opinion it makes no sense to program a script to migrate data (which will end up generating multiple errors and loss of hours) when you can automate a "re-join" with a few lines in powershell or authorize a backup domain (BDC) in univention or nethserver and then terminate zentyal with everything working.
But of course I understand that everyone has their own circumstances and not everything is possible in every situation.

**Comment by Zentyal-Admin** [comment]
I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.

**Comment by peptoniet** [comment]
> I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.

Ok, you got me on a rough day...  This is more than enough.  I you want to fool yourself and your "small team" do as you please.  But do not, I repeat, DO NOT laugh at the rest uf us.

You either have no commitment to your promises or simply do not care.  Now you promise February?

I am so glad I'm out of this.  Even so, I am still a professional, who commits to it's word.

**Y o u  h a v e  n o t  t o u c h e d  t h e  r e p o  f o r  s i x  m o n t h s.**

Are you ready to admit you really "allegedly" work on hidden repos?

Will you stop laughing at us?  There is good hearted people who is still willing to believe you.  Don't you have any compassion or self respect?

And you have the nerve to say that the issue's page of your main repo is not monitored...  What do you do all day, then?

I strongly require you to stop lying to us.

Don't you dare to give any more empty promises to people.

**Comment by FlexibleToast** [comment]
> Are you ready to admit you really "allegedly" work on hidden repos?

It's pretty clear that's what they do. They work in another branch and then merge that branch into master as a release. That's a pretty common strategy. Well, doing it in a hidden repo might not be as common for something open source...

**Comment by Zentyal-Admin** [comment]
> > I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.
> 
> Ok, you got me on a rough day... This is more than enough. I you want to fool yourself and your "small team" do as you please. But do not, I repeat, DO NOT laugh at the rest uf us.
> 
> You either have no commitment to your promises or simply do not care. Now you promise February?
> 
> I am so glad I'm out of this. Even so, I am still a professional, who commits to it's word.
> 
> **Y o u h a v e n o t t o u c h e d t h e r e p o f o r s i x m o n t h s.**
> 
> Are you ready to admit you really "allegedly" work on hidden repos?
> 
> Will you stop laughing at us? There is good hearted people who is still willing to believe you. Don't you have any compassion or self respect?
> 
> And you have the nerve to say that the issue's page of your main repo is not monitored... What do you do all day, then?
> 
> I strongly require you to stop lying to us.
> 
> Don't you dare to give any more empty promises to people.

I just can't imagine why you would think I would lie. We are releasing 8.0 in February. If you'd like to call me a liar you can attempt it in March. All this vitriol. We are just running our little business and building a better product. Sorry it is not happening fast enough for you but it is happening. No need for the hating.

**Comment by peptoniet** [comment]
> > > I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.
> > 
> > 
> > Ok, you got me on a rough day... This is more than enough. I you want to fool yourself and your "small team" do as you please. But do not, I repeat, DO NOT laugh at the rest uf us.
> > You either have no commitment to your promises or simply do not care. Now you promise February?
> > I am so glad I'm out of this. Even so, I am still a professional, who commits to it's word.
> > **Y o u h a v e n o t t o u c h e d t h e r e p o f o r s i x m o n t h s.**
> > Are you ready to admit you really "allegedly" work on hidden repos?
> > Will you stop laughing at us? There is good hearted people who is still willing to believe you. Don't you have any compassion or self respect?
> > And you have the nerve to say that the issue's page of your main repo is not monitored... What do you do all day, then?
> > I strongly require you to stop lying to us.
> > Don't you dare to give any more empty promises to people.
> 
> I just can't imagine why you would think I would lie. We are releasing 8.0 in February. If you'd like to call me a liar you can attempt it in March. All this vitriol. We are just running our little business and building a better product. Sorry it is not happening fast enough for you but it is happening. No need for the hating.

I am happy to see you are now suddenly monitoring this thread... (Strange, I do not need to "monitor".  I just get an email when the thread is updated.  Don't you have an email account...?  I though it was required for opening a github account...)

"I just can't imagine why you would think I would lie".  Easy: because you have done it in the past.  Exactly in your comment on the 13th of October 2022, stating that 8.0 would be released the following May 2023.  Do you remember your words?  Understand now?

I don't hate you.  You are not worth hating.  I hate myself for believing in you for so long.

Vitriol?  How about you start answering about the hidden repos OR why you promise releases in february if the main repo has not been updated in 6 months.  How is this possible...?  Maybe the already small team is shrinking.

I'm autistic.  What is your explanation...?

**Comment by Zentyal-Admin** [comment]
> > > > I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.
> > > 
> > > 
> > > Ok, you got me on a rough day... This is more than enough. I you want to fool yourself and your "small team" do as you please. But do not, I repeat, DO NOT laugh at the rest uf us.
> > > You either have no commitment to your promises or simply do not care. Now you promise February?
> > > I am so glad I'm out of this. Even so, I am still a professional, who commits to it's word.
> > > **Y o u h a v e n o t t o u c h e d t h e r e p o f o r s i x m o n t h s.**
> > > Are you ready to admit you really "allegedly" work on hidden repos?
> > > Will you stop laughing at us? There is good hearted people who is still willing to believe you. Don't you have any compassion or self respect?
> > > And you have the nerve to say that the issue's page of your main repo is not monitored... What do you do all day, then?
> > > I strongly require you to stop lying to us.
> > > Don't you dare to give any more empty promises to people.
> > 
> > 
> > I just can't imagine why you would think I would lie. We are releasing 8.0 in February. If you'd like to call me a liar you can attempt it in March. All this vitriol. We are just running our little business and building a better product. Sorry it is not happening fast enough for you but it is happening. No need for the hating.
> 
> I am happy to see you are now suddenly monitoring this thread... (Strange, I do not need to "monitor". I just get an email when the thread is updated. Don't you have an email account...? I though it was required for opening a github account...)

—I have 27 email addresses coming into one account which generates several thousand emails per week so I don't get to address them all and I use a series of filters to discern what is truly important for me to address. This one just happened to pop up in my main inbox today. 
> 
> "I just can't imagine why you would think I would lie". Easy: because you have done it in the past. Exactly in your comment on the 13th of October 2022, stating that 8.0 would be released the following May 2023. Do you remember your words? Understand now?

—No, you are incorrect. However, that is the date of my birthday should you want to remember it for later this year. But that comment was made by our lead developer, not me. He is not a liar either, he was just overenthusiastic about the delivery date and we made the decision to push the date to Q1 2024. And yes, we should have mentioned this.
> 
> I don't hate you. You are not worth hating. I hate myself for believing in you for so long.

—Harsh. 
> 
> Vitriol? How about you start answering about the hidden repos OR why you promise releases in february if the main repo has not been updated in 6 months. How is this possible...? Maybe the already small team is shrinking.

—Team is same size. I'll ask them about "hidden repos" whatever that is. 
> 
> I'm autistic. What is your explanation...?

—I am a polymath and since selling my investment bank a few years ago I retain several companies that I administrate. My daily work is writing but I will ask the dev team to monitor this more closely should you feel the need to spew more insults in our direction. We could do a much better job of communication. Noted. We do send a newsletter to 28,000 people every month and we also let everyone know what's going on in our social media feeds but I see you would like us to be more active on Github. Which I've never used. But we are on the case, and thank you for waking us from our slumber.

**Comment by peptoniet** [comment]
> > > > > I can assure you the Zentyal is alive and well with 8.0 due to release in February. The development team is based in Spain, admin is based in New York. This is a closed feed and not monitored so apologies for missing questions but the dev team is monitoring bug reports and responding.
> > > > 
> > > > 
> > > > Ok, you got me on a rough day... This is more than enough. I you want to fool yourself and your "small team" do as you please. But do not, I repeat, DO NOT laugh at the rest uf us.
> > > > You either have no commitment to your promises or simply do not care. Now you promise February?
> > > > I am so glad I'm out of this. Even so, I am still a professional, who commits to it's word.
> > > > **Y o u h a v e n o t t o u c h e d t h e r e p o f o r s i x m o n t h s.**
> > > > Are you ready to admit you really "allegedly" work on hidden repos?
> > > > Will you stop laughing at us? There is good hearted people who is still willing to believe you. Don't you have any compassion or self respect?
> > > > And you have the nerve to say that the issue's page of your main repo is not monitored... What do you do all day, then?
> > > > I strongly require you to stop lying to us.
> > > > Don't you dare to give any more empty promises to people.
> > > 
> > > 
> > > I just can't imagine why you would think I would lie. We are releasing 8.0 in February. If you'd like to call me a liar you can attempt it in March. All this vitriol. We are just running our little business and building a better product. Sorry it is not happening fast enough for you but it is happening. No need for the hating.
> > 
> > 
> > I am happy to see you are now suddenly monitoring this thread... (Strange, I do not need to "monitor". I just get an email when the thread is updated. Don't you have an email account...? I though it was required for opening a github account...)
> 
> —I have 27 email addresses coming into one account which generates several thousand emails per week so I don't get to address them all and I use a series of filters to discern what is truly important for me to address. This one just happened to pop up in my main inbox today.
> 
> > "I just can't imagine why you would think I would lie". Easy: because you have done it in the past. Exactly in your comment on the 13th of October 2022, stating that 8.0 would be released the following May 2023. Do you remember your words? Understand now?
> 
> —No, you are incorrect. However, that is the date of my birthday should you want to remember it for later this year. But that comment was made by our lead developer, not me. He is not a liar either, he was just overenthusiastic about the delivery date and we made the decision to push the date to Q1 2024. And yes, we should have mentioned this.
> 
> > I don't hate you. You are not worth hating. I hate myself for believing in you for so long.
> 
> —Harsh.
> 
> > Vitriol? How about you start answering about the hidden repos OR why you promise releases in february if the main repo has not been updated in 6 months. How is this possible...? Maybe the already small team is shrinking.
> 
> —Team is same size. I'll ask them about "hidden repos" whatever that is.
> 
> > I'm autistic. What is your explanation...?
> 
> —I am a polymath and since selling my investment bank a few years ago I retain several companies that I administrate. My daily work is writing but I will ask the dev team to monitor this more closely should you feel the need to spew more insults in our direction. We could do a much better job of communication. Noted. We do send a newsletter to 28,000 people every month and we also let everyone know what's going on in our social media feeds but I see you would like us to be more active on Github. Which I've never used. But we are on the case, and thank you for waking us from our slumber.

To be a polymath not only implies to be dedicated to several disciplines, but also EXCEL in each of them...

Prove me wrong: release 8.0 before the 29th february 2024.

At least, you are lucky: you have one more day in february this year.

**Comment by Pharao01** [comment]
In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out!
I'll test an upgrade from 7.1 and maybe a fresh install from the just I downloaded the ISO this weekend.
Big THANX to the Zentyal team!

**Comment by peptoniet** [comment]
And all the repos were massively updated 4 days ago... wow...

**Comment by peptoniet** [comment]
@Neustradamus I think they are ignoring your on purpose...

"push (@cmds, "mkdir -p $mirror/bid-dns");"
Stil there!!!!
:D :D :D

**Comment by Neustradamus** [comment]
@Pharao01: Thanks for the information!

After several years... a new version!
Congrats to the Zentyal team.

But one problem, my PR has not been merged into Zentyal 8.0 (and previous too):
- https://github.com/zentyal/zentyal/pull/2048

And other PRs too?
- https://github.com/zentyal/zentyal/pulls

**Comment by FlexibleToast** [comment]
> And all the repos were massively updated 4 days ago... wow...

Like I said, they were probably doing work in a branch we couldn't see. Now there is a 8.0 branch as well. It's not uncommon to have your development happen in a different branch and merge it into master/main later.

It worries me that they're keeping it hidden and don't seem to be merging outside PRs. That makes the project less open source and more "source available."

**Comment by peptoniet** [comment]
> > And all the repos were massively updated 4 days ago... wow...
> 
> Like I said, they were probably doing work in a branch we couldn't see. Now there is a 8.0 branch as well. It's not uncommon to have your development happen in a different branch and merge it into master/main later.
> 
> It worries me that they're keeping it hidden and don't seem to be merging outside PRs. That makes the project less open source and more "source available."

Exactly!

**Comment by Neustradamus** [comment]
Note: Sourceforge project has not been updated:
- https://sourceforge.net/projects/zentyal/

**Comment by progamer562** [comment]
dockeerrrr!

**Comment by hudakh** [comment]
> In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out! I'll test an upgrade from 7.1 and maybe a fresh install from the just downloaded ISO this weekend. Big THANX to the Zentyal team!

Can't upgrade yet, only a fresh install is available.

**Comment by Pharao01** [comment]
> > In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out! I'll test an upgrade from 7.1 and maybe a fresh install from the just downloaded ISO this weekend. Big THANX to the Zentyal team!
> 
> Can't upgrade yet, only a fresh install is available.

Yes you're right. I intended to add the new repo but that didn't work out well. I also tried to run the setup in an ubuntu 22.04 LCX, got it installed but failed to join the existing domain - yes I know - it's not intended to be run in a LCX, I know but I had to try.
This weekend maybe I'll do a clean install in a VM.

**Comment by lightsabata** [comment]
> > > In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out! I'll test an upgrade from 7.1 and maybe a fresh install from the just downloaded ISO this weekend. Big THANX to the Zentyal team!
> > 
> > 
> > Can't upgrade yet, only a fresh install is available.
> 
> Yes you're right. I intended to add the new repo but that didn't work out well. I also tried to run the setup in an ubuntu 22.04 LCX, got it installed but failed to join the existing domain - yes I know - it's not intended to be run in a LCX, I know but I had to try. This weekend maybe I'll do a clean install in a VM.

Be careful, with a fresh install, a 7.1 backup is not compatible with a 8.0 system.

**Comment by huda-kh** [comment]
> > > In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out! I'll test an upgrade from 7.1 and maybe a fresh install from the just downloaded ISO this weekend. Big THANX to the Zentyal team!
> > 
> > 
> > Can't upgrade yet, only a fresh install is available.
> 
> Yes you're right. I intended to add the new repo but that didn't work out well. I also tried to run the setup in an ubuntu 22.04 LCX, got it installed but failed to join the existing domain - yes I know - it's not intended to be run in a LCX, I know but I had to try. This weekend maybe I'll do a clean install in a VM.

Can update now! But I'm waiting until I am confident it's not buggy.

**Comment by Pharao01** [comment]
> > > > In order to revive and maybe finally end this thread: Zentyal Server Development Edition 8.0 is out! I'll test an upgrade from 7.1 and maybe a fresh install from the just downloaded ISO this weekend. Big THANX to the Zentyal team!
> > > 
> > > 
> > > Can't upgrade yet, only a fresh install is available.
> > 
> > 
> > Yes you're right. I intended to add the new repo but that didn't work out well. I also tried to run the setup in an ubuntu 22.04 LCX, got it installed but failed to join the existing domain - yes I know - it's not intended to be run in a LCX, I know but I had to try. This weekend maybe I'll do a clean install in a VM.
> 
> Can update now! But I'm waiting until I am confident it's not buggy.

This weekend I tried an upgrade of an existing 7.1 installation which was joined as an additional DC to a Windows DC.
So far it seems that everything is working like a charm. During the upgrade I had some issues with some customized stubs (dns, samba and network). I solved this by removing non-essential ones and reworking the ones needed.
As for now everything seems to run fine.
